tests/MongoDB.Driver.TestHelpers/DriverTestConfiguration.cs

```diff
@@ -140,6 +140,8 @@ public static MongoClientSettings GetClientSettings()
             {
                 serverSelectionTimeoutString = "30000";
             }
+
+            clientSettings.RetryWrites = false;
```

Comment:
Tests that are are not related with `RetryWrites` directly were designed with considering that `RetryWrites` is equal to `false`.